.github/dependabot.yml

```
@@ -9,3 +9,8 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+
+  - package-ecosystem: "github-actions"
+    directory: ".github/actions/setup-buil"
```

```suggestion
    directory: ".github/actions/setup-build"
```
or where is this coming from?